assets/src/scss/editorStyle.scss

```diff
@@ -5,6 +5,7 @@
 @import "~bootstrap/scss/buttons";
 
 @import "base/colors";
+@import "base/tokens";
```

Comment:
I think this `import` is part of [PLANET-7052](https://jira.greenpeace.org/browse/PLANET-7052) to keep it more consistent.